#
# $Id: httpd.conf-2.0.patch,v 1.3 2004/02/14 21:40:46 matt Exp $
#

# Author: Matt Simerson
#
# httpd.conf diffs for Apache 2.0+
# apply this to the stock httpd.conf.
#

*** httpd-std.conf	Fri Nov  7 21:30:13 2003
--- httpd.conf	Sat Nov 22 01:01:22 2003
***************
*** 388,394 ****
  # http://httpd.apache.org/docs-2.0/mod/core.html#options
  # for more information.
  #
!     Options Indexes FollowSymLinks
  
  #
  # AllowOverride controls what directives may be placed in .htaccess files.
--- 389,395 ----
  # http://httpd.apache.org/docs-2.0/mod/core.html#options
  # for more information.
  #
!     Options Indexes FollowSymLinks ExecCGI
  
  #
  # AllowOverride controls what directives may be placed in .htaccess files.
***************
*** 436,442 ****
  # negotiated documents.  The MultiViews Option can be used for the 
  # same purpose, but it is much slower.
  #
! DirectoryIndex index.html index.html.var
  
  #
  # AccessFileName: The name of the file to look for in each directory
--- 437,443 ----
  # negotiated documents.  The MultiViews Option can be used for the 
  # same purpose, but it is much slower.
  #
! DirectoryIndex index.cgi index.shtml index.html index.htm index.html.var
  
  #
  # AccessFileName: The name of the file to look for in each directory
***************
*** 592,597 ****
--- 593,600 ----
  # We include the /icons/ alias for FancyIndexed directory listings.  If you
  # do not use FancyIndexing, you may comment this out.
  #
+ Alias /squirrelmail/ "/usr/local/www/squirrelmail/"
+ Alias /isoqlog/images/ "/usr/local/share/isoqlog/htmltemp/images/"
  Alias /icons/ "/usr/local/www/icons/"
  
  <Directory "/usr/local/www/icons">
***************
*** 905,911 ****
  # To use CGI scripts outside of ScriptAliased directories:
  # (You will also need to add "ExecCGI" to the "Options" directive.)
  #
! #AddHandler cgi-script .cgi
  
  #
  # For files that include their own HTTP headers:
--- 908,916 ----
  # To use CGI scripts outside of ScriptAliased directories:
  # (You will also need to add "ExecCGI" to the "Options" directive.)
  #
! AddHandler cgi-script .cgi
! AddType application/x-httpd-php .php
! AddType application/x-httpd-php-source .phps
  
  #
  # For files that include their own HTTP headers:
